src/govuk/components/accordion/_index.scss

```diff
@@ -44,74 +37,212 @@
   .js-enabled {
     .govuk-accordion {
       // Border at the bottom of the whole accordion
-      border-bottom: 1px solid $govuk-border-colour;
+      border-bottom: $govuk-accordion-bottom-border-width solid $govuk-border-colour;
     }
 
     // Borders between accordion sections
     .govuk-accordion__section {
-      padding-top: 0;
-    }
-
-    // Hide the body of collapsed sections
-    .govuk-accordion__section-content {
-      display: none;
-      @include govuk-responsive-padding(3, "top");
-      @include govuk-responsive-padding(3, "bottom");
-    }
-
-    // Show the body of expanded sections
-    .govuk-accordion__section--expanded .govuk-accordion__section-content {
-      display: block;
+      padding: 0;
     }
 
-    // This is styled to look like a link not a button
-    .govuk-accordion__open-all {
-      @include govuk-font($size: 16);
+    .govuk-accordion__show-all {
+      @include govuk-font($size: 19);
       position: relative;
       z-index: 1;
-      margin: 0;
-      padding: 0;
+
+      margin: 0 0 govuk-spacing(2) 0;
+      padding: 7px 2px 7px 0;
+
       border-width: 0;
+
       color: $govuk-link-colour;
       background: none;
+
       cursor: pointer;
       -webkit-appearance: none;
 
-      @include govuk-link-common;
-      @include govuk-link-style-default;
-
       // Remove default button focus outline in Firefox
       &::-moz-focus-inner {
         padding: 0;
         border: 0;
       }
+
+      &:hover {
+        color: $govuk-accordion-base-colour;
+        background: $govuk-accordion-hover-colour;
+
+        .govuk-accordion__section-toggle-text {
+          color: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron {
+          color: $govuk-accordion-base-colour;
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-hover-colour;
+        }
+      }
+
+      &:focus {
+        @include govuk-focused-text;
+
+        .govuk-accordion-nav__chevron {
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-icon-focus-colour;
+        }
+      }
+    }
+
+    .govuk-accordion__section-heading {
+      @include govuk-font($size: 24, $line-height: 1);
+      // Override browser defaults to ensure consistent element height
+      margin-top: 0; // Override browser default
+      margin-bottom: 0; // Override browser default
+    }
+
+    .govuk-accordion__section-heading-focus-wrapper {
+      margin-bottom: govuk-spacing(1);
+    }
+
+    // Create Chervon icon aligned with text
+    .govuk-accordion-nav__chevron {
+      box-sizing: border-box;
+      display: inline-block;
+
+      position: relative;
+
+      width: 20px;
```

Comment:
Agreed - the `govuk_publishing_components` version of this iteration contains relative values[1] in order to scale as expected, in the context of this contribution it was noted this pattern is perhaps not present within the Design System at present so didn't wish to introduce a change in convention.  This could be worth exploring at a later date for all icons to meet a zoom text-only AC.

[1]https://github.com/alphagov/govuk_publishing_components/commit/192ff22f1332c18d710e42889431b6c92eeda3e1

Reply:
After some thought decided it was more favourable to introduce relative values in this context. [This has been applied here:](https://github.com/alphagov/govuk-frontend/pull/2257/commits/aae5ab9486383189d1450057d1f49bd0e18579e8)

Updated visuals (zoom text only firefox):

![Aug-09-2021 10-58-59](https://user-images.githubusercontent.com/71266765/128689804-37c00076-3309-4bd4-8221-62d24010d36a.gif)

Reply:
Brilliant, thanks @chris-gds 🎉